google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelperTest.java

```diff
@@ -140,11 +150,13 @@ public void testCreateDeployRunnerInvalidSdk() {
 
     assertNull(runner);
     verify(callback, times(1))
-        .errorOccurred("No Cloud SDK was found in the specified directory. /this/path");
+        .errorOccurred(
+            "No Cloud SDK was found in the specified directory. " + path.toString());
```

Comment:
It was suggested to me before that, by containing the integral path name instead of getting it from the reference, unit tests can be more assertive about our desired behaviour.
Doesn't make much difference in this case, though.